%YAML:1.0
---
camera_matrix: !!opencv-matrix
   rows: 3
   cols: 3
   dt: d
   data: [ 1231.4542984211307, 0., 224.24486549875164, 0.,
       1014.062094271705, 312.35206125212261, 0., 0., 1. ]
dist_coeffs: !!opencv-matrix
   rows: 1
   cols: 5
   dt: d
   data: [ -0.90020184211343157, 1.0223536146002052,
       -0.076417127551369965, -0.026747941655827365, -3.5985935722280074 ]
reproj_error: 1.282386638643362
